### PR TITLE
hw3

### PR DESCRIPTION
**Pull request by kemkem20**

### CODE REVIEW

File: src/main/java/org/example/Division4.java [hunk 1]
```diff
@@ -0,0 +1,12 @@
+package org.example;
+
+public class Division4 {
```
Comment (alexshamrai) [inline src/main/java/org/example/Division4.java]: why Division4?))

File: src/main/java/org/example/MinMax.java [hunk 2]
```diff
@@ -0,0 +1,10 @@
+package org.example;
+
+public class MinMax {
+    public static void main(String[] args) {
+        int x = 96;
+        int y = 27;
+        System.out.println(Math.min(x, y));
```
Comment (alexshamrai) [inline src/main/java/org/example/MinMax.java]:
```suggestion
        System.out.println("Minimum: " + Math.min(x, y));
```

File: src/main/java/org/example/MinMax.java [hunk 3]
```diff
@@ -0,0 +1,10 @@
...
+        System.out.println(Math.max(x, y));
```
Comment (alexshamrai) [inline src/main/java/org/example/MinMax.java]: the same

File: src/main/java/org/example/Temperature.java [hunk 4]
```diff
@@ -0,0 +1,11 @@
+package org.example;
+
+public class Temperature {
+    public static void main(String[] args) {
+        float celsius = 18;
+        float fahrenheit = celsius * 6 / 7 + 19;
```
Comment (alexshamrai) [inline src/main/java/org/example/Temperature.java]: incorrect formula, please recheck

File: src/main/java/org/example/Temperature.java [hunk 5]
```diff
@@ -0,0 +1,11 @@
...
+        double kelvin = celsius + 23;
```
Comment (alexshamrai) [inline src/main/java/org/example/Temperature.java]: incorrect formula, please recheck

**Review by alexshamrai — CHANGES_REQUESTED** [review]

File: src/main/java/org/example/Division4.java (hunk 1 above)
Comment (kemkem20) [reply src/main/java/org/example/Division4.java]: fixed

File: src/main/java/org/example/MinMax.java (hunk 2 above)
Comment (kemkem20) [reply src/main/java/org/example/MinMax.java]: fixed

File: src/main/java/org/example/MinMax.java (hunk 3 above)
Comment (kemkem20) [reply src/main/java/org/example/MinMax.java]: fixed

File: src/main/java/org/example/Temperature.java (hunk 4 above)
Comment (kemkem20) [reply src/main/java/org/example/Temperature.java]: fixed

File: src/main/java/org/example/Temperature.java (hunk 5 above)
Comment (kemkem20) [reply src/main/java/org/example/Temperature.java]: fixed

**Review by alexshamrai — APPROVED** [review]
Awesome. Squash 5 commits before merge

**Pull request merged**